### DISCUSSION TITLE
Do I need an API token for this?

### DISCUSSION THREAD

**Discussion by teknowledgist** [community/community#35574] [Apps, API and Webhooks] [answered]
I have some PowerShell code to use the GitHub API to monitor other repositories (not mine) about their latest releases.  That's all it does.  It works fine, but in the off-chance that I want to make a bunch of queries, do I need a Personal Access Token?  

I've read that the API is limited to 60 queries/hour without an access token, but when I go to make one, what scope(s) should it have?  None of the scope items seem to apply (at least not in language that is obvious to me).  

Thanks!

**Comment by fadyat** [comment, marked as answer]
Yes, unauthenticated API requests to GitHub are capped at 60 per hour. Authenticated API requests 5000 per hour.

I think you can create a token without any scopes. You don't need any 
You don't need them if you're trying to get data from someone else's public repositories.
